dex/src/main/scala/com/wavesplatform/dex/settings/utils/ConfigOps.scala

```diff
@@ -32,6 +33,31 @@ object ConfigOps {
       }
     }
 
+    def filterKeys(part: String): Config = {
+      var oc = config
+
+      def filter(c: Config, b: String = ""): Unit = {
```

Comment:
`filter` should return `Config`, then you don't need to `var`, e.g.:
```scala
def filter(c: Config, b: String = ""): Config = {
 // ...
 oc.withoutPath(p) // returns Config
}
```

Also you have to use `asScala.foldLeft(c) { ... }` with this approach.